.dockerignore

```diff
@@ -0,0 +1,4 @@
+./web/node_modules
+./web/dist
+.git*
```

Comment:
is this for `.github/workflows/main.yml` ?

Reply:
it's mostly for `.git/` which otherwise takes time to copy as we run `COPY . .`
the wildcard may not be necessary